1-exercises/B-todo-list/script.js

```diff
@@ -1,12 +1,34 @@
 function todoList(todos) {
-  // Write your code here...
+  let ulEl = document.createElement("ulEl");
+  ulEl.setAttribute("id", "list");
+  let headingEl = document.createElement("h1");
+  headingEl.innerHTML = "Todo list";
+  let content = document.querySelector("#content");
+  content.appendChild(headingEl);
+  for (task of todos) {
+    // console.log(todo);
+    let liEl = document.createElement("liEl");
+    liEl.setAttribute("class", "listItem");
+    ulEl.appendChild(liEl);
+    liEl.innerHTML = task.todo; // show the list of items in objects
+
+    // function created to put line through each item
+    function setListDone(liEl) {
+      liEl.addEventListener("click", function () {
+        this.classList.toggle("done");
+      });
+    }
+
+    setListDone(liEl); // Calling the function
```

Comment:
It is good to write a separate function to extract all related actions into once place for better code organisation. For this case, unless you have extra logics to be enforced on decide how the event listener is added, you may just register the event listener directly.